frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
Ohk, no it's not hosted. 
No problem, let's merge this then you can check it. It's correct from my side and according to design we have to do this.